hazelcast/src/main/java/com/hazelcast/client/config/ClientConfig.java

```diff
@@ -1005,13 +1008,40 @@ public ClientConfig setSqlConfig(ClientSqlConfig sqlConfig) {
         return this;
     }
 
+    /**
+     * Returns the Alto configuration.
+     *
+     * @return the Alto configuration
+     * @since 5.3
+     */
+    @Beta
+    @Nonnull
+    public ClientAltoConfig getAltoConfig() {
+        return altoConfig;
+    }
+
+    /**
+     * Sets the Alto configuration.
+     *
+     * @param altoConfig Alto config to set
+     * @return configured {@link ClientConfig} for chaining
+     * @throws IllegalArgumentException if the {@code altoConfig} is {@code null}
```

Comment:
Yes, it is not done for most of the documentation, but I believe it is a better practice to do it. It might be too much to change all the existing documentation, but I think we can at least introduce the new documentation with all possible exceptions listed.